test_env_vars.yaml

```diff
@@ -152,35 +153,19 @@ pip_packages:
   # packages: expected packages from the Cachito response
   # purl: PURL of the package
   # dep_purls: PURLs if dependencies
+  # files_to_ignore: source files what we can ignore with checkings
   with_deps:
     repo: https://github.com/cachito-testing/cachito-pip-with-deps.git
     ref: 83b387568b6287f6829403cff1e1377b0fb2f5d8
     pkg_managers: ["pip"]
+    files_to_ignore: ["PKG-INFO", "aiowsgi.egg-info", ".gitignore", ".travis.yml"]
     expected_files:
-      setup.cfg: https://raw.githubusercontent.com/cachito-testing/cachito-pip-with-deps/master/setup.cfg
-      requirements.txt: https://raw.githubusercontent.com/cachito-testing/cachito-pip-with-deps/master/requirements.txt
-    expected_deps_files:
-      - pip/aiowsgi/aiowsgi-0.7.tar.gz
-      - pip/external-appr/appr-external-sha256-ee6a0a38bed8cff46a562ed3620bc453141a02262ab0c8dd055824af2829ee5c.zip
-      - pip/github.com/quay/appr/appr-external-gitcommit-58c88e4952e95935c0dd72d4a24b0c44f2249f5b.tar.gz
-    dependencies:
-      - dev: false
-        name: "appr"
-        replaces: null
-        type: "pip"
-        version: "git+https://github.com/quay/appr.git@58c88e4952e95935c0dd72d4a24b0c44f2249f5b"
-      - dev: false
-        name: "appr"
-        replaces: null
-        type: "pip"
-        version: "https://github.com/quay/appr/archive/37ff9a487a54ad41b59855ecd76ee092fe206a84.zip#egg=appr&cachito_hash=sha256:ee6a0a38bed8cff46a562ed3620bc453141a02262ab0c8dd055824af2829ee5c"
-      - dev: false
-        name: "aiowsgi"
-        replaces: null
-        type: "pip"
-        version: "0.7"
-    packages:
-      - dependencies:
+      app: https://github.com/cachito-testing/cachito-pip-with-deps/tarball/83b387568b6287f6829403cff1e1377b0fb2f5d8
+      deps/pip/aiowsgi/aiowsgi-0.7.tar.gz : https://github.com/gawel/aiowsgi/archive/0.7.tar.gz
```

Comment:
Can we fetch the dependency from the same resource Cachito is fetching it? i.e., Cachito is fetching it from PyPI, as we have `aiowsgi==0.7` in the requirement file. If we fetch it from PyPI as well, we will not need the files_to_ignore field at all.

You can use the json PyPI API to fetch the proper URL:

`https://pypi.org/pypi/aiowsgi/0.7/json` will show that an `sdist` for 0.7 is available at `https://files.pythonhosted.org/packages/f4/3d/1933776c5215c61e38968fedd73c41251e8752736e1cd9fbb73db44ff4e1/aiowsgi-0.7.tar.gz`

Reply:
If we still need this files_to_ignore field, it should list files to be ignored per dependency.

Reply:
We don't need it anymore :)